### ISSUE TITLE
Make app free for dev stores

### ISSUE THREAD

**Issue by faisal-saeed-brainx** [ohmybrew/laravel-shopify#315]
I have created a paid plan in db with 7 days trial.

On dev store, after completing 7 days trial it redirects to billing and shows popup to upgrade plan.

Please help me in order to make free app for all dev stores.

Plan values are:

id => 1
type => 1
name -> Basic
price => 0.99 
capped_amount => NULL
terms => Basic terms
trial_days => 7
test => 1
on_install =>1

**Comment by Kyon147** [comment]
You can use the AfterAuthenticateJob to check the store plan type and then set the grandfathered column in the shop's table to true.

This would then mean all dev stores are "free". There is a list somewhere (can't find it atm) where it tells you all the different shop types.

**Comment by ohmybrew** [comment]
Hi @faisal-saeed-brainx 

@Kyon147  is correct, a contributor made an article several months ago detailing how to do this.

https://github.com/ohmybrew/laravel-shopify/wiki/Free-Access-Based-on-Shopify-Plan

Given this is not a bug or issue I am currently closing this, if you feel there is a bug, please reopen. Thank you.

**Comment by faisal-saeed-brainx** [comment]
Hi @ohmybrew @Kyon147 

Thanks for your support. I have applied this approach and it is  working fine.

I have to discuss 1 test scenario with you, When we installed the app on a dev store, the AfterAuthenticateJob updates the grandfathered value as true. then after that each time it skips the api call due to grandfatherd value is true. After that user upgrades the store plan to other paid plan.

My question is for above scenario:
Now the store plan has been changed on the shopify_plan, on which it should be charge. But the grandfathered column is still true and will work as free.
